suite-common/message-system/config/config.v1.json

```diff
@@ -50,6 +50,37 @@
                 "context": { "domain": "accounts.coinjoin" }
             }
         },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=24.5.2",
+                        "mobile": "!",
+                        "web": ">=24.5.2"
+                    }
+                }
+            ],
```

Comment:
does not need to be here

Reply:
I would leave it there for better consistency with other messages